package_manifest.yaml

```diff
@@ -721,18 +721,10 @@ foreman_plugin_packages:
   children:
     ruby_plugin_packages: {}
   hosts:
-    ansible-collection-theforeman-operations: {}
-    ansiblerole-foreman_scap_client: {}
-    ansiblerole-insights-client: {}
     ansible-runner: {}
-    ansible-collection-theforeman-foreman: {}
-    cjson: {}
     foreman-discovery-image-service: {}
```

Comment:
This is a weird one. I wonder if the discovery image should be its own category.

Reply:
a clear maybe (I first moved it to the "other" group too, and then was like "nah, this is a special one")